File: exs/mundo_1/java/005.java
/*
Desafio 005
Problema: Faça um programa que leia um número inteiro e mostre na tela
        o seu sucessor e antecessor.
Resolução do problema:
*/
import java.util.Scanner;

class Main{
    public static void main(String[] args){
        Scanner scan = new Scanner(System.in);
        System.out.print("Antecessor e sucessor\nDigite um número: ");
        int num = scan.nextInt();
        System.out.printf("%d - %d - %d\n", num-1, num, num+1);
    }
}

File: exs/mundo_1/golang/005.go
/*
	Desafio: 005
	Problema: Faça um programa que leia um número inteiro 
	e mostre na tela o seu sucessor e seu antecessor
*/

package main 
import "fmt"

func main() {
	var number int
	var sucessor int 
	var predecessor int
	fmt.Print("Digite um número: ")
	fmt.Scan(&number)
	sucessor = number + 1
	predecessor = number - 1
	fmt.Println("Antecessor:", predecessor, "Número:", number, "Sucessor:" ,sucessor)
}